charts/kube-master/templates/cloud-controller-manager.yaml

```diff
@@ -50,27 +50,64 @@ spec:
             items:
               - key: openstack.config
                 path: openstack.config
+        {{- if .Values.controllerManager.rejectProxy.enabled }}
```

Comment:
Maybe we should remove the `controllerManager` part of this value path: `{{- if .Values.rejectProxy.enabled }}`

Reply:
I removed this and changes values.